microsoft-edge/webview2/release-notes/index.md

```diff
@@ -330,9 +661,9 @@ The following APIs have been promoted from Experimental to Stable in this Prerel
    * [ICoreWebView2FileSystemHandle::get_Permission](/microsoft-edge/webview2/reference/win32/icorewebview2filesystemhandle?view=webview2-1.0.2646-prerelease&preserve-view=true#get_permission)
 
 * [ICoreWebView2ObjectCollection](/microsoft-edge/webview2/reference/win32/icorewebview2objectcollection?view=webview2-1.0.2646-prerelease&preserve-view=true)
-   * [ICoreWebView2ObjectCollection::RemoveValueAtIndex](/microsoft-edge/webview2/reference/win32/icorewebview2objectcollection?view=webview2-1.0.2646-prerelease&preserve-view=true#removevalueatindex)
+   * [ICoreWebView2ObjectCollection::get_Kind](/microsoft-edge/webview2/reference/win32/icorewebview2objectcollection?view=webview2-1.0.2646-prerelease&preserve-view=true#get_kind)<!-- todo: test, b/c anchor section seems to not exist -->
```

Comment:
same as the [Comment in newer section](https://github.com/MicrosoftDocs/edge-developer/pull/3218/files#r1704263765).  This link is like a 404; it defaults to top of the API Ref webpage for objcollection type, b/c no such "get" anchor section exists in Ref docs.

this instance of the get_Kind link is in section:
1.0.2646-prerelease -- June 19, 2024 -- The following APIs have been promoted from Experimental to Stable in this Prerelease SDK.
@peiche-jessica @victorhuangwq

Reply:
removed bc "There shouldn't be a ICoreWebView2ObjectCollection::get_Kind, it's a bug in the script"